document/CoreFormat.mdk

```diff
@@ -1743,13 +1729,138 @@ A `codeFlow` object may contain a property named `tags`, whose value is a tag se
 A `codeFlow` object may contain a property named `properties`, whose value is a property bag.
 This allows tools to include information about the code flow that is not explicitly specified in the SARIF format.
 
+## `stack` object { #stack }
+
+A `stack` object describes a single call stack.
+A call stack is a sequence of nested function calls, each of which is referred to as a stack frame.
+
+### `message` property { #stack-message }
+
+A `stack` object may contain a property named `message` whose value is a string
+containing a message relevant to this call stack.
+
+### `frames` property { #stack-frames }
+
+A `stack` object shall contain a property named `frames` whose value is an array of one or more `stackFrame` objects (&sect;[#stackFrame]).
+This array shall include every function call in the stack for which the tool has information,
+and the entries that are present shall occur in order with the innermost call first and the outermost call last.
+The entries in this array need not be unique.
+It is possible for the same frame to occur multiple times if the calls stack includes a recursion. 
+
+~ IsoNote
+It is possible that the analysis tool will not have location information for every frame in the call stack.
+This might happen if, for example, application code for which location information is available calls into operating system code
+for which location information is not available, which in turn calls back into application code.
+~
+
+### `tags` property { #stack-tags }
+
+A `stack` object may contain a property named `tags`, whose value is a tag set.
+
+### `properties` property { #stack-properties }
+
+A `stack` object may contain a property named `properties`, whose value is a property bag.
+This allows tools to include information about the stack that is not explicitly specified in the SARIF format.
+
+## `stackFrame` object { #stackFrame }
+
+### General { #stackFrame-General }
+
+A `stackFrame` object describes a single stack frame within a call stack (&sect;[#stack]).
+
+### `message` property { #stackFrame-message }
+
+A `stackFrame` object may contain a property named `message` whose value is a string
+containing a message relevant to this stack frame.
+
+### `uri` property { #stackFrame-uri }
+
+A `stackFrame` object may contain a property named `uri` whose value is a string
+containing the URI of the source code file to which this stack frame refers.
+
+~ IsoNote
```

Comment:
ok, I think the simplest thing is to go by your initial comment and just omit the notes on `uri` and `line`.